### ISSUE TITLE
Right clicking status effect in upper right corner does not eliminate the effect.

### ISSUE THREAD

**Issue by KLSteinwand** [ernieayala/ernies-modern-layout#131]
VTT: 0.8.8
PF2e: 2.9.1.6656
Modern UI: 0.3.10

When applying a status effect through either the token such as Flat-Footed, Prone, Grabbed, or drag and dropping from the chat menu via an action card, there will be one effect unable to removed by right clicking the icon. The status effect can still be removed through the conditions available on the token HUD itself or through the character sheet. If multiple effects are shown at the same time, they will intermittently clear out, however there is still one that remains. Sadly I am unable to provide more information than that as the console will not register the mouse click, even through attempting with multiple mice to attempt a possible hardware issue.

**Comment by ernieayala** [comment]
@KLSteinwand I only have latest PF2e system installed and my mod installed. Is this working as expected? I find no code in my area that hinders the clicking and since I only do CSS it wouldn't dismiss anything automatically. Might be a PF2e setting or another mod?

https://user-images.githubusercontent.com/9342233/126262960-df016454-d95c-4793-89a4-9218aaeaa594.mp4

**Comment by ernieayala** [comment]
Another thing I noticed is that if I add the status Confused - it also adds Flat Footed. Which you can not dismiss as it's tied to Confused. When I dismiss Confused it will also clear Flat Footed.

**Comment by KLSteinwand** [comment]
I have tried with other module combinations to try and see if there is any confliction but were not able to see any. There are some conditions that are combined, such as grabbed which also applies the flat-footed and immobilized conditions they will all clear out. However if they a single condition or modifier, they will not clear. So conditions such as stunned or enfeebled that last for varying rounds, modifiers that are single instance such as cover or flat footed, or spell effects that have a limited duration, they will not fully clear. For instance I had a player who casted shield and ended up having 3 instances of it remaining, even though it did recognize that the spell effect no longer applied and therefore did not modify the character statistics. I was able to clear it via the character sheet though as intended. After reloading the application which I initially thought would be the fix, I was able to clear 2 of the 3 shield effects but not the last one, even after multiple reloads within the application and closing the application to re-open it. Anything under the top box seems to clear as it should when being right clicked, with only the second giving some slight issue as an intermittent occasion. But in terms of the top condition, it does recognize through the PF2e system that it is there, it does apply the appropriate effects as intended, and does show the listed condition when hovered over, but its as if that section of GUI is not allowing for manipulation.
